ports/servoshell/test.rs

```diff
@@ -85,3 +89,97 @@ fn test_argument_parsing_special() {
     assert_eq!(url.query(), None);
     assert_eq!(url.fragment(), None);
 }
+
+#[test]
+fn url_should_resolve_in_commad_line() {
+    embedder_traits::resources::set_for_tests();
+    let input = "resources/public_domains.txt";
+    let cwd = "../../";
+    env::set_current_dir(&cwd).expect("Failed to set current directory");
+
+    let result = get_default_url(Some(input.to_string()));
```

Comment:
nit: to_owned(), or ideally remove these and change the parameter to `Option<&str>`

Reply:
went with Option<&str> in [e801366](https://github.com/servo/servo/pull/30148/commits/e801366322dbf5f8b317af36231f70bfab41d9ec) :)